src/Chapter1.hs

```diff
@@ -508,7 +509,12 @@ branches because it is an expression and it must always return some value.
   satisfying the check will be returned and, therefore, evaluated.
 -}
 closestToZero :: Int -> Int -> Int
-closestToZero x y = error "closestToZero: not implemented!"
+closestToZero x y =
+    if absX < absY then
+      absX
+    else absY
```

Comment:
Actually, you need to return the value itself instead of the absolute value. In this case, we lose the sign. But it is easy to fix 👍🏼